networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -1,45 +1,74 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="albo-sts-cluster"]
-= Preparing for the AWS Load Balancer Operator on a cluster using the AWS {sts-full}
+= Installing the AWS Load Balancer Operator on a cluster that uses AWS {sts-short}
 :context: albo-sts-cluster
 
 toc::[]
 
-You can install the AWS Load Balancer Operator on a cluster that uses STS. Follow these steps to prepare your cluster before installing the Operator.
+You can install the {aws-first} Load Balancer Operator on a cluster that uses the {sts-first}. Follow these steps to prepare your cluster before installing the Operator.
 
-The AWS Load Balancer Operator relies on the `CredentialsRequest` object to bootstrap the Operator and the AWS Load Balancer Controller. The AWS Load Balancer Operator waits until the required secrets are created and available.
+The {aws-short} Load Balancer Operator relies on the `CredentialsRequest` object to bootstrap the Operator and the {aws-short} Load Balancer Controller. The {aws-short} Load Balancer Operator waits until the required secrets are created and available.
+
+[id="{context}_prerequisites"]
+== Prerequisites
+
+* You installed the {oc-first}.
+
+* You know the infrastructure ID of your cluster. You can issue the following command in your CLI to output this ID:
```

Comment:
Give the "why" before the "how" and use consist term for running commands.

Rewrite as: "To display this ID, run the following command in your CLI:"

Reply:
:heavy_check_mark: